addons/mail/static/src/backend_components/activity_list_view_item/activity_list_view_item.xml

```diff
@@ -18,7 +18,7 @@
                     <a t-if="activityListViewItem.fileUploader" class="fa fa-upload" title="Upload file" role="img" t-on-click="activityListViewItem.onClickUploadDocument"/>
                     <a t-else="" class="o_ActivityListViewItem_markAsDone fa fa-check-circle" t-on-click="activityListViewItem.onClickMarkAsDone" title="Mark as done" role="img" aria-label="Mark as done" t-ref="markDoneButton"/>
                     <!-- TODO sign override to change the button -->
-                    <!-- TODO approvals override to change stuff -->
+                    <t xpath-target="approvals-activity-list-item"/>
```

Comment:
I m sad to let go the `xpath-target` idea since it's more explicit in wath it does. Targeting a random class that happen to be the good one seam fragile.